src/gui/include/gui/gui.h

```diff
@@ -90,6 +90,14 @@ class Painter
              (b == other.b) &&
              (a == other.a);
     }
+
+    static const Color copyWithAlpha(const Color& color, int a)
```

Comment:
Maybe this should just be a constructor?

Color(const Color& color, int a);

Reply:
that's a better solution, done